src/table/TableBodyWrapper.jsx

```diff
@@ -68,56 +72,96 @@ const TableBodyWrapper = ({
     addSelectionListeners(selectionsAPI, selDispatch, setShouldRefocus);
   }, []);
 
+  useEffect(() => {
+    setSrNotation('');
+
+    setTimeout(() => {
+      setSrNotation(getSrNotation());
+    }, MAX_NOTATION_DELAY);
+  }, [focusedCellCoord, selState]);
+
+  const getSrNotation = () => {
+    if (focusedCellCoord.toString() == '0,0') return '';
```

Comment:
I guess this for the entire first row?

Reply:
you mean we should check it for the entire first row? 
currently it is for the very first cell

Reply:
✅ Sure, Fixed
I'm now checking it with `startsWith('0')`

Reply:
why not just `focusedCellCoord[0] === 0`

Reply:
yes that could be done as well 
✅ Fixed